Test failure kernel.inprocess

```
Test group: kernel.inprocess
F.......
======================================================================
FAIL: Does pylab work in the in-process kernel?
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/IPython/testing/decorators.py", line 225, in skipper_func
    return f(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/IPython/kernel/inprocess/tests/test_kernel.py", line 49, in test_pylab
    self.assert_('matplotlib' in msg['content']['data'])
AssertionError: False is not true
    'False is not true' = self._formatMessage('False is not true', "%s is not true" % safe_repr(False))
>>  raise self.failureException('False is not true')

-------------------- >> begin captured logging << --------------------
IPython.kernel.inprocess.ipkernel: DEBUG:
*** MESSAGE TYPE:execute_request***
IPython.kernel.inprocess.ipkernel: DEBUG:    Content: {u'code': u'%pylab', u'silent': False, u'user_variables': [], u'store_history': True, u'user_expressions': {}, u'allow_stdin': True}
   --->

IPython.kernel.inprocess.ipkernel: DEBUG: {'parent_header': {u'date': datetime.datetime(2014, 9, 26, 11, 48, 27, 263401), u'username': u'beccamorgan92', u'session': u'07e688ec-88dc-4011-a9d1-7fc50398e5ca', u'msg_id': u'4f602414-2f3f-46f6-8903-e2f8623b5ee2', u'msg_type': u'execute_request'}, 'msg_type': u'execute_reply', 'msg_id': '58c2ccab-dfc6-44f5-9213-6449b6de06c8', 'content': {u'status': u'ok', u'execution_count': 1, u'user_variables': {}, u'payload': [], u'user_expressions': {}}, 'header': {'date': datetime.datetime(2014, 9, 26, 11, 48, 28, 701177), 'username': u'beccamorgan92', 'session': u'8786d191-9eee-4b1e-82dd-19da74c0e33d', 'msg_id': '58c2ccab-dfc6-44f5-9213-6449b6de06c8', 'msg_type': u'execute_reply'}, 'tracker': <zmq.sugar.tracker.MessageTracker object at 0x109223710>, 'metadata': {'dependencies_met': True, 'engine': u'474aa60e-7008-45e5-97ec-e21146ec614c', 'status': u'ok', 'started': datetime.datetime(2014, 9, 26, 11, 48, 27, 267768)}}
--------------------- >> end captured logging << ---------------------

----------------------------------------------------------------------
Ran 8 tests in 1.655s

FAILED (failures=1)

______________________________________________________________________
Test suite completed for system with the following information:
IPython version: 2.2.0
IPython commit : d87a8cd (installation)
IPython package: /usr/local/lib/python2.7/site-packages/IPython
Python version : 2.7.8 (default, Sep 14 2014, 19:04:32) [GCC 4.2.1 Compatible Apple LLVM 5.1 (clang-503.0.40)]
sys.executable : /usr/local/opt/python/bin/python2.7
Platform       : Darwin-13.4.0-x86_64-i386-64bit

Tools and libraries available at test time:
   casperjs curses cython jinja2 matplotlib numpy oct2py pexpect pygments pymongo qt requests rpy2 sphinx sqlite3 tornado zmq

Status: ERROR - 1 out of 1 test groups failed (kernel.inprocess). Took 3.056s.

You may wish to rerun these, with:
  iptest kernel.inprocess
```

--- a/IPython/kernel/inprocess/tests/test_kernel.py
+++ b/IPython/kernel/inprocess/tests/test_kernel.py
@@ -9,6 +9,7 @@ import unittest
 from IPython.kernel.inprocess.blocking import BlockingInProcessKernelClient
 from IPython.kernel.inprocess.manager import InProcessKernelManager
 from IPython.kernel.inprocess.ipkernel import InProcessKernel
+from IPython.kernel.tests.utils import assemble_output
 from IPython.testing.decorators import skipif_not_matplotlib
 from IPython.utils.io import capture_output
 from IPython.utils import py3compat
@@ -33,8 +34,8 @@ class InProcessKernelTestCase(unittest.TestCase):
         """Does %pylab work in the in-process kernel?"""
         kc = self.kc
         kc.execute('%pylab')
-        msg = get_stream_message(kc)
-        self.assertIn('matplotlib', msg['content']['text'])
+        out, err = assemble_output(kc.iopub_channel)
+        self.assertIn('matplotlib', out)
 
     def test_raw_input(self):
         """ Does the in-process kernel handle raw_input correctly?
@@ -63,21 +64,6 @@ class InProcessKernelTestCase(unittest.TestCase):
         kc = BlockingInProcessKernelClient(kernel=kernel)
         kernel.frontends.append(kc)
         kc.execute('print("bar")')
-        msg = get_stream_message(kc)
-        self.assertEqual(msg['content']['text'], 'bar\n')
+        out, err = assemble_output(kc.iopub_channel)
+        self.assertEqual(out, 'bar\n')
 
-#-----------------------------------------------------------------------------
-# Utility functions
-#-----------------------------------------------------------------------------
-
-def get_stream_message(kernel_client, timeout=5):
-    """ Gets a single stream message synchronously from the sub channel.
-    """
-    while True:
-        msg = kernel_client.get_iopub_msg(timeout=timeout)
-        if msg['header']['msg_type'] == 'stream':
-            return msg
-
-
-if __name__ == '__main__':
-    unittest.main()

